source-code/sdk-js/src/helpers/ast.test.ts

```diff
@@ -3,7 +3,7 @@ import { parse } from "acorn"
 import { wrapVariableDeclaration, insertAst } from "./ast.js"
 // @ts-ignore
 import { generate } from "astring"
-import type { ArrowFunctionExpression, Program } from "estree"
+import type { ArrowFunctionExpression, Program, Node } from "estree"
```

Comment:
For tests we can use the `!` assertion. We don't need th cast. Please revert those changes

Reply:
Will do